awxkit/awxkit/api/pages/api.py

```diff
@@ -48,6 +49,7 @@
     ('Inventory', 'Host'),
     ('Inventory', 'Label'),
     ('WorkflowJobTemplateNode', 'WorkflowApprovalTemplate'),
+    ('Credential', 'CredentialInputSource'),
```

Comment:
You probably don't want this here.

Reply:
won't this allow to export the credential input sources when imoprting a credential with an input source?

Reply:
No.  As I explained before, `DEPENDENT_EXPORT` doesn't quite work with how this relationship is being exposed in the API.  You'll have to leave this out, and in order to correctly export a user will need to do both `--credentials` and `--credential_input_sources`.